examples/log-macro-demo.rs

```diff
@@ -0,0 +1,25 @@
+extern crate sentry_rs;
+#[macro_use]
+extern crate log;
+
+use sentry_rs::models::SentryCredentials;
+use sentry_rs::Sentry;
+use sentry_rs::logging::SentryLogger;
+use std::env;
+
+fn main() {
+  let dsn = "https://XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX:YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY@sentry.io/XX";
```

Comment:
Can we change this to getting the DSN from the environment similar to other examples? This allows some testing I do easier (as I can validate directly in sentry).

Reply:
Sounds good, I must have missed that example, will take a look shortly.